consai_examples/tests/test_geometry_tools.py

```diff
@@ -2,38 +2,97 @@
 import math
 import consai_examples.geometry_tools as tool
 
+from consai_msgs.msg import State2D
+
 def test_trans_class():
     
-    trans = tool.Trans([0, 0], float(math.pi / 2))
-    p_trans = trans.transform([0, 1]) 
-    assert math.isclose(p_trans[0], 1, abs_tol=0.01)
-    assert math.isclose(p_trans[1], 0, abs_tol=0.01)
+    pose1 = State2D(x=0.0, y=0.0)
+    pose2 = State2D(x=0.0, y=1.0)
+    trans = tool.Trans(pose1, float(math.pi / 2))
```

Comment:
推奨される使い方をテストに記入してほしいです。
```suggestion
    trans = tool.Trans(pose1, tool.get_angle(pose1, pose2))
```

Reply:
書いた